worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/CommaSeparatedValuesConverter.java

```diff
@@ -73,7 +86,17 @@ public Component describeAcceptableArguments() {
     public List<String> getSuggestions(String input, InjectedValueAccess context) {
         String lastInput = Iterables.getLast(COMMA.split(input), "");
         assert lastInput != null;
-        return delegate.getSuggestions(lastInput, context);
+        List<String> suggestions = delegate.getSuggestions(lastInput, context);
+        if (input.length() > lastInput.length()) {
```

Comment:
Can we make this condition `input.contains(",")` instead? (Not sure if that's valid Java tbh, but you know what I mean)